### ISSUE TITLE
easing-tests.js is out of date

### ISSUE THREAD

**Issue by egirard** [web-platform-tests/wpt#15483]
Looking at easing-tests.js in [wpt](https://github.com/web-platform-tests/wpt/blob/master/web-animations/resources/easing-tests.js), [moz](https://dxr.mozilla.org/mozilla-central/source/testing/web-platform/tests/web-animations/resources/easing-tests.js), and [chromium](https://cs.chromium.org/chromium/src/third_party/blink/web_tests/external/wpt/web-animations/resources/easing-tests.js?q=easing-tests.js&sq=package:chromium&g=0&l=1), it seems that mozilla's is more recent than the others. It seems to have been updated in https://phabricator.services.mozilla.com/D9309 but those changes weren't upstreamed.

@birtles, @BorisChiou can you advise?

**Comment by egirard** [comment]
More specifically, there's a section starting at line 44 that includes:
 ```
  {
    desc: 'frames function',
    easing: 'frames(5)',
    easingFunction: framesTiming(5)
  },
```

**Comment by BorisChiou** [comment]
It seems the wpt bot didn't upstream the update of `easing-tests.js`. I can create a PR to update it.

**Comment by BorisChiou** [comment]
@jgraham do you have any idea to re-trigger the wpt bot for the patch (https://phabricator.services.mozilla.com/D9309)?

**Comment by birtles** [comment]
Yeah, it looks like the sync bot picked up the changes in parts 2 and 8 of that patch series but not part 1:

  https://github.com/web-platform-tests/wpt/pull/13868/commits

There may have been a merge conflict apparently:

  https://bugzilla.mozilla.org/show_bug.cgi?id=1502788#c1

**Comment by birtles** [comment]
I followed up with @jgraham on this. He suspects a misresolved merge conflict and is going to look into it.

**Comment by birtles** [comment]
Fixed by a4bfa25bfc35e6dd8aabf9bc5af714bf3d70d712.
